cli/service/service.go

```diff
@@ -169,18 +170,27 @@ func LogMac(instanceId string, tailing bool) {
 
 func LogLinux(instanceId string, tailing bool) {
 	msgPrinter := i18n.GetMessagePrinter()
+	// Determine the system log file based on linux distribution
+	sysLogPath := "/var/log/messages"
+	_, err := os.Stat("/etc/redhat-release")
+	if os.IsNotExist(err) {
+		_, err := os.Stat("/etc/centos-release")
+		if os.IsNotExist(err) {
+			sysLogPath = "/var/log/syslog"
+		}
+	}
 	// The requested service is running, so grab the records from syslog for this service.
-	file, err := os.Open("/var/log/syslog")
+	file, err := os.Open(sysLogPath)
 	if err != nil {
-		cliutils.Fatal(cliutils.NOT_FOUND, msgPrinter.Sprintf("/var/log/syslog could not be opened or does not exist: %v", err))
+		cliutils.Fatal(cliutils.NOT_FOUND, msgPrinter.Sprintf("%v could not be opened or does not exist: %v", sysLogPath, err))
```

Comment:
The go text tools will pick up this message change and will automatically adjust the gotext files.

Reply:
Thanks @dabooz!  It is good to know that gotext translation are handled by tools via IBM donation.   I was actually wondering how that would be handled, until I saw an example like this:  https://github.com/open-horizon/anax/pull/2031
I've reverted all changes to gotext*.json files.

Reply:
I also further updated pkg/rpm/README to remove the limitation statement.